plugins/common/docker/stats_helpers.go

```diff
@@ -6,6 +6,7 @@ import (
 	"github.com/docker/docker/api/types/container"
 )
 
+// CalculateCPUPercentUnix calculate CPU usage (for Unix, in percentages)
 func CalculateCPUPercentUnix(previousCPU, previousSystem uint64, v *container.StatsResponse) float64 {
```

Comment:
Do we really need all those exported?

Reply:
These methods are used by two plugins (`docker` and `ecs`). 
It seems reasonable to me that they ended up in the `common` packages - this way one plugin (`ecs`) doesn't import another plugin (`docker`).

Reply:
Oh boy I completely missed the `common` part here... Then of course this needs to be exported. Sorry for the noise!